src/yunohost/backup.py

```diff
@@ -152,6 +152,21 @@ def _clean_tmp_dir(retcode=0):
         'hooks': {},
     }
 
+    # Initialize backup list
+    backup_csv_path = os.path.join(tmp_dir, 'backup.csv')
+    try:
+        backup_csv = open(backup_csv_path, 'a')
+    except:
+        logger.error(m18n.n('backup_csv_creation_failed'))
+
+    def add_into_list(source,dest):
+        try:
+            if dest.endswith("/"):
+                dest=dest+os.path.basename(source)
+            backup_csv.write("{0}\t{1}\n".format(source,dest))
```

Comment:
We need a way to read this csv from bash post_backup_create hooks. May be a CSV with delimiter could be more difficult to read...